config/locales/js-en.yml

```diff
@@ -32,6 +32,11 @@ en:
       hide: "Hide"
       loading: "Loading ..."
     close_popup_title: "Close popup"
+
+    clipboard:
+      browser_error: "Your browser doesn't seem to support copying to clipboard. Please copy the selected text manually."
```

Comment:
Minor:

> Your browser doesn't ~~seem to~~ support copying to clipboard.

I personally also tend to weaken my messages ("I am not sure, but it looks like..."), but thinking about it: No that stupid browser does not work as our application wants it to work, so we can tell that.